src/calendar/calendar-month.style.tsx

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { TextStyleHelper } from "../text/helper";
+import { VariantMonth } from "./calendar-month";
+
+interface MonthCellProps {
+    $variant: VariantMonth;
+}
+
+export const MonthPickerContainer = styled.div`
+    display: grid;
+    grid-template-columns: repeat(2, 13.5rem);
+    grid-template-rows: repeat(6, 2.5rem);
+    gap: 0.5rem 1rem;
```

Comment:
space between columns seems wider in figma
```suggestion
    gap: 0.5rem 2.5rem;
```